roles/hdfs-standalone/tasks/datanode.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Configure Hadoop HDFS DataNode
+  sudo: yes
+  template: src={{ item }}.xml.j2 dest={{ hdfs_conf_dir }}/{{ item | replace("namenode-", "") }}.xml
```

Comment:
why does this task use `name=value` syntax, and the rest use `name: value` syntax?